apps/condo/domains/news/components/NewsForm/InputStep/index.tsx

```diff
@@ -0,0 +1,408 @@
+}
+
+export type NewsItemDataType = {
+    type: string
+    validBefore?: string
+    title: string
+    body: string
+}
+
+type NewsItemSharingFormProps = {
+    sharingAppData?: {
+        app: B2BApp
+        id: string
+        newsSharingConfig: B2BAppNewsSharingConfig
+        ctx: any
+    }
+    onSubmit: (SharingAppValues: SharingAppValuesType) => void
+    onSkip: (SharingAppValues: SharingAppValuesType) => void
+
+    initialValues: SharingAppValuesType | undefined
+
+    newsItemData: NewsItemDataType
+}
+
+export type BaseNewsFormProps = {
+    organizationId: string
+    initialValues?: Partial<INewsItem>
+    & Partial<{
+        newsItemScopes: Array<INewsItemScope>
+        hasAllProperties: boolean
+        sendPeriod: SendPeriodType
+        properties?: IProperty[]
+    }>
+    templates: TemplatesType
+    totalProperties: number
+    autoFocusBody?: boolean
+}
+
+type InputStepProps = NewsItemSharingFormProps & BaseNewsFormProps & {
+    form: FormInstance
+    scope: ScopeType
+    setScope: React.Dispatch<React.SetStateAction<ScopeType>>
+    isSharingStep: boolean
+    selectedProperty: {
+        loading: boolean
+        objs:  IProperty[]
+    }
+    initialPropertyIds: string[]
+    initialFormValues: Record<string, unknown>
+}
+
+export const FormContainer: React.FC = ({ children }) => {
+    return (
+        <Col span={24} css={BIG_MARGIN_BOTTOM_STYLE}>
+            <Row gutter={BIG_HORIZONTAL_GUTTER}>
+                {children}
+            </Row>
+        </Col>
+    )
+}
+
+export const InputStep: React.FC<InputStepProps> = ({
+    form,
+    scope,
+    onSkip,
+    onSubmit,
+    setScope,
+    templates,
+    isSharingStep,
+    newsItemData,
+    initialValues,
+    autoFocusBody,
+    sharingAppData,
+    organizationId,
+    totalProperties,
+    selectedProperty,
+    initialPropertyIds,
+    initialFormValues,
+}
+) => {
+    const { app: sharingApp, id: sharingAppId } = sharingAppData ?? { id: null, app: null }
+    const { id, newsSharingConfig } = sharingApp ?? { id: null, newsSharingConfig: null }
+    const { title, body } = newsItemData
+    const { loading: selectedPropertiesLoading, objs: selectedProperties } = selectedProperty
+
+    const appName = newsSharingConfig?.name
+    const appPreviewUrl = newsSharingConfig?.previewUrl
+
+    const iFramePreviewRef = useRef(null)
+
+    const isCustomForm = !!newsSharingConfig?.customFormUrl && isSharingStep
+    const isCustomPreview = !!newsSharingConfig?.previewUrl && isSharingStep
+
+    const viewNewsSharingSubmit = isSharingStep
+
+    const { breakpoints } = useLayoutContext()
+    const isMediumWindow = !breakpoints.DESKTOP_SMALL
+
+    const intl = useIntl()
+    const SelectAddressPlaceholder = intl.formatMessage({ id: 'global.select.address' })
+    const NextStepShortMessage = intl.formatMessage({ id: 'pages.condo.news.steps.skipLabelShort' })
+    const NextStepMessage = intl.formatMessage({ id: 'pages.condo.news.steps.nextStep' })
```

Comment:
we have an unspoken rule that translations are declared as high as possible in the component